package.json

```diff
@@ -36,23 +45,23 @@
   "homepage": "https://github.com/orbitdb/orbit-db-identity-provider#readme",
   "devDependencies": {
     "@babel/core": "^7.5.5",
-    "@babel/plugin-syntax-object-rest-spread": "^7.2.0",
-    "@babel/plugin-transform-modules-commonjs": "^7.5.0",
-    "@babel/plugin-transform-runtime": "^7.5.5",
-    "@babel/preset-env": "^7.5.5",
+    "@babel/preset-env": "^7.20.2",
     "@babel/runtime": "^7.5.5",
     "babel-loader": "^8.0.6",
     "fs-extra": "^10.0.0",
     "is-node": "^1.0.2",
     "json-loader": "^0.5.7",
-    "key-did-provider-ed25519": "^1.0.0",
-    "key-did-resolver": "^1.2.1",
-    "localstorage-level-migration": "~0.1.0",
+    "key-did-provider-ed25519": "^2.0.1",
+    "key-did-resolver": "^2.3.0",
+    "level-js": "^6.1.0",
+    "levelup": "^5.1.1",
+    "localstorage-level-migration": "^0.0.2",
     "mocha": "^8.4.0",
     "mocha-headless-chrome": "^3.1.0",
+    "orbit-db-keystore": "^1.0.1-ae215fb.0",
```

Comment:
This should be in "dependencies" since it's pulled in by [src/identities.js](https://github.com/orbitdb/orbit-db-identity-provider/pull/93/files#diff-1b8278b094b8f2a8fe23f03e691224f46febd22370984455f5daaac13f6c4405R6) and [src/identity-providers/orbitdb.js](https://github.com/orbitdb/orbit-db-identity-provider/pull/93/files#diff-ab9e4f5378d9f950d63ff61eaa80aa1436018f25429c27ee40bafa0f21fbce9eR2).